pages/wishlist/wishlist.css

```diff
@@ -0,0 +1,27 @@
+
+.wishlist-container{
+    display: grid;
+    grid-template-columns:repeat(4, 1fr);
+}
+
+
+
+@media screen and (max-width: 1100px) {
+    .wishlist-container{
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+    }
+}
+
+@media screen and (max-width: 600px) {
+    .wishlist-container{
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+    }
+}
+@media screen and (max-width: 420px) {
+    .wishlist-container{
+        display: grid;
+        grid-template-columns: 1fr;
+    }
```

Comment:
I can see your media queries but it is not fully responsiove. Hope you work on it later.

Reply:
yes its not fully responsive yet
i will work on that